sdk/nodejs/Makefile

```diff
@@ -27,6 +27,12 @@ ensure:: yarn.ensure node.ensure .ensure.phony
 	yarn install --frozen-lockfile
 	@touch .ensure.phony
 
+format:: ensure
+	yarn rome format --write .
+
+format_ci:: ensure
+	yarn rome ci .
```

Comment:
Just to confirm: this will still verify that code that is poorly formatted fails in CI, right? That's my read of this comment and the Rome docs, but I want to confirm.

Reply:
Yes, `rome ci` sets the status code to non-zero if there's formatting violations, which fails CI. You can run it locally too; it doesn't run exclusively in CI environments.